week03/html/about.html

```diff
@@ -0,0 +1,64 @@
+<link href="../css/about.css" rel="stylesheet" /> 
+<link rel="icon" type="image/png" sizes="16x16" href="../src/billyLogo.png"/>
+
+<title>Lee Gabin</title>
+
+<body>
+
+    <!--상단바-->
+    <header id="header">
+        <div class="navbar">
+        <div class="nav1">
+        <a href="../html/home.html" class="logo">
+        <img src="../src/mainLogo.png" height="70px" />
+    </a>
+    </div>
+    <div class="nav2"></div>
+    <div class="nav3">
+    <ul>
+        <li><a href="../html/home.html">Home</a></li>
+        <li><a href="../html/about.html" style="color: black;">About</a></li>
+        <li><a href="../html/project.html">Project</a></li>
+        <li><a href="../html/activity.html">Activity</a></li>
+        <li><a href="../html/contact.html">Contact</a></li>
+    </ul>
+    </div>
+    </div>
+    </header>
+
+    <!--내용-->
+    <div class="aboutme">
+        <div class="title">About</div>
+        <div class="space"></div>
+        <div class="content">
+        <img class="BLog" src="../src/BLog.png" height="350px" />
+        <table border="0">
+            <tr>
+                <td><a class="th">Name</a></td>

```

Comment:
1. `table` 태그에는 다음과 같은 태그들이 있습니다.
   - `tr`: table row
   - `th`: table header
   - `td`: table data

```suggestion
                <th>Name</th>
```